frontend/src/components/Login.js

```diff
@@ -64,16 +64,17 @@ function Login() {
         </label>
         <br />
         <input
+          name="password"
           type="password"
           className="shadow appearance-none border border-red-500 rounded w-full py-2 px-3 text-gray-700 mb-3 leading-tight focus:outline-none focus:shadow-outline"
           id="password"
           placeholder="******************"
```

Comment:
it should be consistent, multiple asterisk symbols don't say much :/
`placeholder="password"`